polar2grid/writers/hdf5.py

```diff
@@ -0,0 +1,292 @@
+import os
+import sys
+import logging
+import pkg_resources
+import satpy
+import h5py
+import xarray
+import dask
+
+from itertools import groupby
+from datetime import datetime as datetime
+from satpy.writers import Writer
+from satpy.readers.yaml_reader import FileYAMLReader
+from satpy.writers import compute_writer_results
+
+from trollsift import Parser
+
+from pyresample.geometry import SwathDefinition
+from polar2grid.utils.legacy_compat import convert_p2g_pattern_to_satpy
+from polar2grid.writers.geotiff import NUMPY_DTYPE_STRS, NumpyDtypeList,\
+    str_to_dtype, int_or_float
+
+from dask.diagnostics import ProgressBar
+
+from polar2grid.core.script_utils import (
+    setup_logging, rename_log_file, create_exc_handler)
+
+USE_POLAR2GRID_DEFAULTS = bool(int(os.environ.setdefault("USE_POLAR2GRID_DEFAULTS", "1")))
+
+LOG = logging.getLogger(__name__)
+
+# reader_name -> filename
+DEFAULT_OUTPUT_PATTERN = "{platform_name}_{sensor}_{start_time:%Y%m%d_%H%M%S}.h5"
```

Comment:
With current master I think this has to be `DEFAULT_OUTPUT_FILENAMES = {None: "{platform_name}..."}``